libavformat/Makefile

```diff
@@ -493,6 +493,7 @@ OBJS-$(CONFIG_RSD_DEMUXER)               += rsd.o
 OBJS-$(CONFIG_RPL_DEMUXER)               += rpl.o
 OBJS-$(CONFIG_RSO_DEMUXER)               += rsodec.o rso.o pcm.o
 OBJS-$(CONFIG_RSO_MUXER)                 += rsoenc.o rso.o rawenc.o
+OBJS-$(CONFIG_RTC_MUXER)                 += rtcenc.o avc.o
```

Comment:
should add http.o if rtcenc.o will use http.o functions.

Reply:
Add http.o to the list of dependencies, and also add srtp.o.